exercises/calculator.cpp

```diff
@@ -10,3 +10,33 @@
   Multiplication: 8
   Division: 2
 */
+
+#include<iostream>
+
+using namespace std;
+
+int main(){
+  float num1;
+  float num2;
+
+  cout<<"Insert the first number: "<<endl;
+  cin>>num1;
+
+  cout<<"Insert the second number: "<<endl;
+  cin>>num2;
+
+  float sum,diff,mult,div;
+
+  sum = num1 + num2;
+  diff = num1 - num2;
+  mult = num1 * num2;
+  div = num1 / num2;
+
+  cout<<"SUM: "<<sum<<endl;
+  cout<<"Difference: "<<diff<<endl;
+  cout<<"Multiplication: "<<mult<<endl;
+  cout<<"Division: "<<div<<endl;
```

Comment:
Questa parte potrebbe leggermente essere più leggibile nel seguente modo:
```suggestion
  cout << "SUM: " << sum << endl;
  cout << "Difference: " << diff << endl;
  cout << "Multiplication: " << mult << endl;
  cout << "Division: " << div << endl;
```